src/python/test/test_dxclient.py

```diff
@@ -6210,6 +6210,260 @@ def test_upload_resources(self):
         # Verify that the bundled depends appear in the same folder.
         self.assertEqual(resources_file_describe['folder'], '/subfolder')
 
+    def _build_check_resources(self, app_dir, args=""):
+        # create applet and get the resource_file id
+        new_applet = json.loads(run("dx build -f --json " + args + " " + app_dir))
+        applet_describe = dxpy.api.applet_describe(new_applet["id"])
+        resources_file = applet_describe['runSpec']['bundledDepends'][0]['id']['$dnanexus_link']
+        id1 = dxpy.api.file_describe(resources_file)['id']
+
+        # download resources tar and extract it and check lib/libz.so is copied to resources/libz.so
```

Comment:
Is this comment current? I don't see any assertion to this effect, if that's what you meant to say.